packages/neos-ui-validators/src/NotEmpty/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import I18n from '@neos-project/neos-ui-i18n';
+
+const NotEmpty = value => {
+    if (value === null || value === '' || value === []) {
+        return <I18n id="content.inspector.validators.notEmptyValidator.isEmpty" fallback="The value can't be empty"/>;
+    }
```

Comment:
I would personally always return at least `null` as a fallback, `undefined`  is dangerous since it can be overwritten